docs/manage/data/query.md

```diff
@@ -0,0 +1,128 @@
+1. [Configure data capture](/services/data/configure-data-capture/) for at least one component, such as a sensor.
+   Only components that capture tabular data support data query.
+1. [Configure cloud sync](/services/data/configure-cloud-sync/), and sync data to the Viam app.
+   When you are able to [view your data in the Viam app](/manage/data/view/), you are ready to proceed.
+
+In addition, if you intend to directly query tabular data from an MQL-compatible client, you must also [configure direct data query](/services/data/configure-data-query/).
+
+## Query tabular data in the Viam app
+
+Once you have synced tabular data to the Viam app, you can run SQL or MQL queries against your synced data from the **Query** subtab under the **Data** tab in the Viam app.
+You must have the [owner](/manage/fleet/#permissions) role in order to query data in the Viam app.
+
+1. Navigate to the [Viam app](https://app.viam.com), select the **Data** tab, then click the **Query** subtab.
+
+1. Select either `SQL` or `MQL` from the **Query mode** dropdown menu on the right-hand side, then enter your query using the respective syntax for the language you have selected in the text area at the top of your screen.
+   Click **Run query** when ready to perform your query and return matching results.
+
+## Query tabular data using the API
+
+Once you have synced tabular data to the Viam app, you can use the following data management API methods in your `go` code to query tabular data using either SQL or MQL:
+
+- Use the [`TabularDataBySQL`](/services/data/#tabulardatabysql) method to query synced tabular data using SQL.
+- Use the [`TabularDataByMQL`](/services/data/#tabulardatabymql) method to query synced tabular data using the [MQL (MongoDB Query Language)](https://www.mongodb.com/docs/manual/tutorial/query-documents/).
+
+These APIs are currently implemented only in the [Go SDK](https://pkg.go.dev/go.viam.com/rdk).
+
+## Query tabular data directly from a compatible client
+
+Once you have synced tabular data to the Viam app, you can directly query that data from an MQL-compatible client, such as [`mongosh`](https://www.mongodb.com/docs/mongodb-shell/) or [Compass](https://www.mongodb.com/docs/compass/current/).
+
+1. If you haven't already, [configure direct data query](/services/data/configure-data-query/#configure-data-query) to be able to use this query mode.
+
+1. Open your chosen MQL-compatible client an connect to the Viam organization's MongoDB Atlas instance.
+   You can use any client that is capable of connecting to a MongoDB Atlas instance, including the [`mongosh` shell](https://www.mongodb.com/docs/mongodb-shell/), [MongoDB Compass](https://www.mongodb.com/docs/compass/current/), and many third-party tools.
+   To connect, use the `hostname` you determined when configuring direct data query, and structure your username in the following format:
+
+   ```sh {class="line-numbers linkable-line-numbers"}
+   db-user-<YOUR-ORG-ID>
```

Comment:
Is this helpful? Don't you just get the full thing back as you need it from the command? It confuses me a little whether I need to do somethign to change it or not

Reply:
I guess maybe it's clear if you follow along

Reply:
We [do not](https://github.com/viamrobotics/docs/pull/2197#discussion_r1393335312) get the username back from the `configure` command, in fact.

Reply:
It sounds like it may be useful for me to return the db user name in the CLI command or in the API. Can file a ticket.

Reply:
Thanks! I think that would help users, especially as `hostname` returns similarly.